packages/salesforcedx-utils-vscode/src/telemetry/telemetry.ts

```diff
@@ -238,12 +257,28 @@ export class TelemetryService {
         );
       }
     });
+    this.maybeWriteToLocalFile(commandName || '', {
+      ...properties,
+      ...measurements
+    });
   }
 
   public sendException(name: string, message: string) {
     this.validateTelemetry(() => {
       this.reporter!.sendExceptionEvent(name, message);
     });
+    this.maybeWriteToLocalFile(name, { message });
+  }
+
+  private maybeWriteToLocalFile(
+    eventName: string,
+    properties: { [key: string]: string | number }
+  ) {
+    if (this.localReporter) {
+      this.localReporter.writeToFile(eventName, {
+        ...properties
+      });
```

Comment:
Should we try/catch and only log in the catch around this?  having actual errors thrown out of this service due to local logging would be a bummer